src/main/java/Parser.java

```diff
@@ -0,0 +1,181 @@
+
+                System.out.println("Now you have " + arrayList.size() + " task(s) in the list");
+                System.out.println("--------------------------");
+
+            } else if (input.contains("event") && !(input.equals("event"))) {
+                System.out.println("--------------------------");
+                System.out.println("Got it. I've added this task: ");
+                String[] parts = input.split("/",2);
+                String part1 = parts[0];
+                String part2 = parts[1];
+
+                // e.g. event project meeting /at 02/12/2019 2-4pm
+                if (part2.contains("/")) {
+                    String dateString = part2.substring(3,13);
+                    String timeString = part2.substring(13);
+                    String temp = dateString.substring(6) + "-" + dateString.substring(3, 5) + "-" + dateString.substring(0, 2);
+                    LocalDate xx = LocalDate.parse(temp);
+                    String f = xx.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + timeString ;
+
+                    Task task = new Event(part1.substring(6), f);
+                    new TaskList().addToList(arrayList, task);
+                } else {
+                    Task task = new Event(part1.substring(6), part2.substring(3));
+                    new TaskList().addToList(arrayList, task);
+                }
+                System.out.println("Now you have " + arrayList.size() + " task(s) in the list");
+                System.out.println("--------------------------");
+
+            }  else if (input.contains("delete") && !(input.equals("delete"))) {
+
+                int deletedNumber = Integer.parseInt(input.substring(7));
+                try {
+                    if (deletedNumber > arrayList.size()) {
+                        throw new DukeException("☹ OOPS!!! Sorry item no found :-(");
+                    } else {
+                        System.out.println("--------------------------");
+                        System.out.println("Noted. I've removed this task: ");
+                        new TaskList().deleteFromList(arrayList, deletedNumber);
+                        System.out.println("Now you have " + arrayList.size() + " task(s) in the list");
+                        System.out.println("--------------------------");
+                    }
+                } catch (DukeException e) {
+                    System.out.println("--------------------------");
+                    System.out.println(e.getMessage());
+                    System.out.println("--------------------------");
+                }
+
+            } else if (input.contains("find") && !(input.equals("find"))) {
+                System.out.println("--------------------------");
+                System.out.println("Here are the matching tasks in your list:");
+
+                ArrayList<Task> tt = new ArrayList<>();
+
+                int counterOne = 1;
+
+                for (Task ttt :arrayList) {
```

Comment:
Naming of variable should be meaningful. Perhaps you could change to it to myTask instead, similar to the one above(ArrayList<Task> tt)?